### PR TITLE
IDEMPIERE-6301 : Add line height to info tab content on about window

### PR DESCRIPTION
**Pull request by uthadehikaru**
https://idempiere.atlassian.net/browse/IDEMPIERE-6301

# Pull Request Checklist

- [x] My code follows the [code guidelines](https://wiki.idempiere.org/en/Contributing_to_Trunk) of this project
- [x] My code follows the best practices of this project
- [x] I have performed a self-review of my own code
- [x] My code is easy to understand and review. 
- [x] I have checked my code and corrected any misspellings
- [x] In hard-to-understand areas, I have commented my code.
- [x] My changes generate no new warnings
### Tests
- [x] I have tested the direct scenario that my code is solving
- [x] I checked all collaterals that can be affected by my changes, and tested other potential affected scenarios
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added unit tests that prove my fix is effective or that my feature works
### Documentation
- [ ] I have made corresponding changes to the documentation as follows:
- - [ ] New feature (non-breaking change which adds functionality): I have created the New Feature page in the project wiki explaining the functionality and how to use it. If relevant, I have committed sample data to the core seed to have usable examples in GardenWorld.
- - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected): I have documented the change in a clear way that everyone in the community can understand the impact of the change.
- - [ ] Improvement (improves and existing functionality): This documentation is needed if the improvement changes the way the user interacts with the system or the outcome of a process/task changes. If it is just, for instance, a performance improvement, documentation might not be needed. 
- [ ] The changed/added documentation is in the project wiki (not privately-hosted pdf files or links pointing to a company website) and is complete and self-explanatory.

### CODE REVIEW

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/AboutWindow.java [hunk 1]
```diff
@@ -367,7 +367,7 @@ protected Tabpanel createInfo() {
 		LayoutUtils.addSclass("about-info-panel", div);
 		div.setParent(tabPanel);
 		ZKUpdateUtil.setHeight(div, "100%");
-		div.setStyle("overflow: auto;");
+		div.setStyle("overflow: auto; line-height: 1.3em;");
```
Comment (CarlosRuiz-globalqss) [inline org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/AboutWindow.java:370]: Hi @uthadehikaru - I was testing this and I like the change, but I find it even better if we make the font a bit littler, like this:

```
div.setStyle("overflow: auto; line-height: 1.3em; font-size: 12px");
```

do you like it too?

Comment (uthadehikaru) [reply org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/AboutWindow.java]: looks good

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/AboutWindow.java [hunk 2]
```diff
@@ -367,7 +367,7 @@ protected Tabpanel createInfo() {
 		LayoutUtils.addSclass("about-info-panel", div);
 		div.setParent(tabPanel);
 		ZKUpdateUtil.setHeight(div, "100%");
-		div.setStyle("overflow: auto;");
+		div.setStyle("overflow: auto; line-height: 1.3em; font-size: 12px");
```
Comment (hengsin) [inline org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/AboutWindow.java:370]: I think this setStyle call should be remove. The settings should goes into the "about-info-panel" css class above instead.

**Review by hengsin — CHANGES_REQUESTED** [review]
need to update org.idempiere.zk.iceblue_c.theme/src/metainfo/zk/lang-addon.xml too.

**Comment by uthadehikaru** [comment]
> @uthadehikaru - I tried that yesterday but didn't work in my tests - the font size was still shown as 14px

fixed. please check again

File: org.idempiere.zk.iceblue_c.theme/src/web/theme/iceblue_c/css/fragment/about.css.dsp [hunk 3]
```diff
@@ -1,6 +1,9 @@
 .about-trace-panel {
 }
 .about-info-panel {
+    overflow: auto;
+    line-height: 1.3em;
```
Comment (hengsin) [inline org.idempiere.zk.iceblue_c.theme/src/web/theme/iceblue_c/css/fragment/about.css.dsp:5]: I think a line-height of 1.5em is more readable.

**Review by hengsin — CHANGES_REQUESTED** [review]

**Review by hengsin — APPROVED** [review]
tested

**Pull request merged**